README.md

```diff
@@ -31,22 +33,47 @@ const users = {
     }
 };
 
-const validate = function (request, username, password, callback) {
+const validate = async (request, username, password, h) => {
```

Comment:
What's the point of `h` if you're not using it ? Should there be an example with it being used ?

Reply:
I'll remove it from here as it's not used. Don't need to show all functionality in the example.

Reply:
Is there any case where you would use it ?

Reply:
Yes, you can set the return object `response` property to a custom response. Previously this was passed in the error position but it doesn't seem appropriate for it to be `throw`n.

You would use `h` to do something like:

```js
return { response: h.redirect('http://hapijs.com') };
```

Reply:
May be worth of an example then ?

Reply:
Sure, added to the README: https://github.com/hapijs/hapi-auth-basic/pull/61/commits/d6396b9b05bd1c686c61f51ea5a557cb20860d97